common/errors.go

```diff
@@ -1567,6 +1567,20 @@ var NewErrEndpointMissingData = func(cause error) error {
 	}
 }
 
+type ErrEndpointDataOutOfRangeForFullNode struct{ BaseError }
+
+const ErrCodeEndpointDataOutOfRangeForFullNode = "ErrCodeEndpointDataOutOfRangeForFullNode"
```

Comment:
```suggestion
const ErrCodeUpstreamNodeTypeMismatch = "ErrUpstreamNodeTypeMismatch"
```
* I used "Endpoint" in errors that are returned from node itself (not a logic within erpc itself). So "Upstream" is probably a better one as it's because erpc itself thinks this node type is not good one (without even sending a request).
* If we use NodeType error, then we can have one or two fields in "details" of the error like (expectedNodeType: "archive" actualNodeType: "full") then this same error can be used in other places
* The word "Code" doesn't need to appear in the code string